templates/nvidia.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: kubeaddons.mesosphere.io/v1alpha1
 kind: Addon
 metadata:
   name: nvidia
-  namespace: kubeaddons
+  namespace: kube-system
```

Comment:
let's use a different namespace name (its own namespace name) otherwise I'll let them under kubeaddos or another namespace.

This might imply to update the charts with their own custom roles.

Reply:
This is done to maintain backwards compatibility:

Going forward the Addon will deploy to the namespace in the Addon metadata, as opposed to what's in the spec.

I would assert that changing which namespace the underlying Addon components are deployed to (as opposed to the Addon resource itself) is out of the scope of this PR.

Reply:
There is not backwards compatibility here, we can deployed whenever we want